erpnext_com/www/docs/user/manual/en/human-resources/employee_skill_map.md

```diff
@@ -0,0 +1,25 @@
+# Employee Skill Map
+
+Employee Skill Map is a record which helps your Organization to track your employee's skill-sets and trainings. It can be used to rate each skill of the employee and track their growth after each internal trainings. This can be used at the time of appraisals.
```

Comment:
```suggestion
Employee Skill Map is a record which helps your organization track your Employee's skill sets and trainings. It can be used to rate each skill of the Employee and track their growth after each internal training. This data can be used at the time of appraisals.
```